frontend/src/metabase/setup/components/SettingsPage/SettingsPage.tsx

```diff
@@ -30,13 +32,22 @@ const STEP_COMPONENTS: Partial<
 
 export const SettingsPage = (): JSX.Element => {
   const steps = useSelector(getSteps);
+  const isEmbeddingUseCase = useSelector(getIsEmbeddingUseCase);
+  const SELECT_WIDTH = 160;
```

Comment:
would prefer to see this as `10rem`;

Reply:
https://github.com/metabase/metabase/commit/9e1673eca0d08242bbbf8e6e4bfee587a164b575#diff-f6e2494aeaff91bc27a3e350d97d0a8764ac9bf39b912b01b7e82e52afd710dcR36